### DISCUSSION TITLE
response	SystemError("<class 'pyodbc.Error'> returned a result with an error set")

### DISCUSSION THREAD

**Discussion by skuhikar-kc** [mkleehammer/pyodbc#1172] [Q&A]
Please first make sure you have looked at:

* Documentation: https://github.com/mkleehammer/pyodbc/wiki
* Other issues

### Environment

To diagnose, we usually need to know the following, including version numbers.  On Windows, be
sure to specify 32-bit Python or 64-bit:

- Python: 
- pyodbc: 
- OS: 
- DB:
- driver:

### Issue

Often it is easiest to describe your issue as "expected behavior" and "observed behavior".

**Comment by v-chojas** [comment]
If you'd provide just a little bit more info, we might actually be able to give you some sort of answer.

**Reply by skuhikar-kc** [reply]
We are getting this error on filters when someone tries to filter data on charfilters. This error is coming from unhandelled exception. The requested path is 
'/alltransactions/?user_id=612348f4cdbdfb00712246a5&role=staff&partner=Ennobl%20Career%20Management%20Services'
However, if i try the same in local, i get filtered data.

**Traceback**
•	/tmp/8db14f8ce6b0308/antenv/lib/python3.8/site-packages/django/core/handlers/exception.py, line 55, in inner 
55.	                response = get_response(request)
56. 
log_data | {'request_method': 'GET', 'request_path': '/alltransactions/?user_id=612348f4cdbdfb00712246a5&role=staff&partner=Ennobl%20Career%20Management%20Services'}
-- | --
request | <WSGIRequest: GET '/alltransactions/?user_id=612348f4cdbdfb00712246a5&role=staff&partner=Ennobl%20Career%20Management%20Services'>
response | SystemError("<class 'pyodbc.Error'> returned a result with an error set")
self | <wallet.wallet_middleware.RequestLogMiddleware object at 0x73daeae20640>

Can you please help, what is going wrong in filters?

**Reply by v-chojas** [reply]
Given that you have not provided __any__ of the initial information we asked for, nor does it seem like you have done anything to isolate the problem directly to pyODBC, we cannot provide any assistance here.

**Reply by skuhikar-kc** [reply]
what information is needed in this regard?
